sympy/ntheory/factor_.py

```diff
@@ -1525,6 +1525,92 @@ def antidivisor_count(n):
         divisor_count(n) - divisor_count(n, 2) - 5
 
 
+def max_pow_in_number(x,y):
```

Comment:
this can be written as `multiplicity(y, x)`:

```
Help on function multiplicity in module sympy.ntheory.factor_:

multiplicity(p, n)
    Find the greatest integer m such that p**m divides n.

    Examples
    ========

    >>> from sympy.ntheory import multiplicity
    >>> from sympy.core.numbers import Rational as R
    >>> [multiplicity(5, n) for n in [8, 5, 25, 125, 250]]
    [0, 1, 2, 3, 3]
    >>> multiplicity(3, R(1, 9))
    -2
```